src/hooks/useShowNotification.ts

```diff
@@ -0,0 +1,45 @@
+import { NotificationData, showNotification } from "@mantine/notifications";
+import { MessageKeys, NestedKeyOf, useTranslations } from "next-intl";
+import { useCallback } from "react";
+
+type NotificationType = "success" | "error";
+
+interface ShowNotificationProps extends NotificationData {
+  type: NotificationType;
+}
+
+const colorsMap: Record<NotificationType, string> = {
+  success: "green",
+  error: "red",
+};
+
+const titleKeysMap: Record<
+  NotificationType,
+  MessageKeys<IntlMessages, NestedKeyOf<IntlMessages>>
+> = {
+  success: "popup.title.success",
+  error: "popup.title.error",
+};
+
+const POPUP_AUTO_CLOSE_TIMEOUT = 5000;
```

Comment:
@gordoney Do we have some standard way to define style for constants? Should it _camelCase_ or _UPPER_CASE_?

Reply:
@Dzmitry-Yaniuk 
If you don't mind, I prefer UPPER_CASE